specs/0118-notification-center.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: Notification Center
+- Created: 2024-03-12
+- Associated Issue: 
+
+# Summary
+
+Introduce a Notification Center feature to make Pontoon the best tool to send communications to localizers directly, allowing sending of targeted in-app notifications and emails within Pontoon itself.
+
+# Motivation
+
+Currently, we can only send in-app notifications with limited targeting capabilities (selected locales within a single product). For any other communications targeting a subset of users we need to rely on engineers to extract email addresses and send emails to the targeted recipients. This feature would allow administrators of a Pontoon instance managing localization projects to send notifications or emails to targeted sets of recipients with more refined criteria such as locales, projects, roles, recency of activity, and level of activity.
+
+# Feature explanation
+
+## Notification Center
+
+This feature has the ability to compose a message with a subject and body, send a message as either an in-app notification or email, and target a set of recipients based on commonly used criteria.
+
+### Required functionality:
+
+- Title: `Notification Center`
+- Header: `Message type`
+  - Selects the type of message to be sent
+  - Checkboxes: `Email`, `Notification` (multiselect, default: both boxes empty)
```

Comment:
```suggestion
  - Checkboxes: `Email`, `Notification` (default: both checkboxes deselected)
```

Checkboxes are by nature multiselect (there's no concept of group, that would become a radio buttons group).

Same comment applies where this formula is repeated later on.